README.md

```diff
@@ -0,0 +1,34 @@
+<h1>Baza de date de filme
+  <h2>Interogari / Actiuni:
+  <h3>Client:
+    <ul>
+      <li>Inregistrare cont
+      <li>Log in
+      <li>Afisare informatii film/serial
+      <li>Adaugare film in lista de filme vazute
+      <li>Adaugare film in watchlist
+      <li>Rate la film
+      <li>Afisare lista de filme vazute
+      <li>Afisare watchlist
+      <li>Afisare liste personalizate de filme
+    </ul>
+  <h3>Client premium:
+    <ul>
+      <li>Creare liste personalizate de filme
+      <li>Review la film
+    </ul>
+  <h3>Admin:
+    <ul>
+      <li>Log in
+      <li>Adaugare film in baza de date
+      <li>Stergere film din baza de date
+      <li>Updatare informatii film
+    </ul>
+  <h2>Obiecte:
+    <ul>
+      <li>User (client, client premium, admin)
+      <li>Movie (+ Show)
+      <li>MovieList
+      <li>Services
```

Comment:
[Fix] services si main nu intra in cele minim 8 entitati din cerinta. Astfel ai doar 6 obiecte de tip POJO